### DISCUSSION TITLE
Starting a process that will outlive shell on Linux

### DISCUSSION THREAD

**Discussion by defufna** [PowerShell/PowerShell#16014] [Q&A] [answered]
Is it possible to start a process that will outlive Powershell on Linux? I tried start-process, but if I terminate powershell, it takes all it's children with it. Bash and friends offer disown command for these situations, but I can't find equivalent in pwsh. On windows, it behaves differently, child processes outlive shell by default...

**Comment by kasini3000** [comment, marked as answer]
on issue ,you can find answer:
```powershell
Start-Process -FilePath '/usr/bin/nohup' -ArgumentList "command arg1  arg2"
```

**Reply by defufna** [reply]
Hey thanks, didn't know about nohup!
